config/Make.rules.Darwin

```diff
@@ -7,7 +7,7 @@ supported-platforms             = macosx iphoneos iphonesimulator
 xcframeworks                    = Ice IceDiscovery IceLocatorDiscovery
 
 macosx_ar                       = libtool
-macosx_cppflags                 = -mmacosx-version-min=12.00
+macosx_cppflags                 = -mmacosx-version-min=14.00
```

Comment:
On my system (macos 14.00), libmcpp.a from homebrew is 14.00 and you get a warning if you try to incorporate it into a 12.00 executable:

```
c++  -o bin/slice2cpp  src/slice2cpp/build/macosx/obj/CPlusPlusUtil.o src/slice2cpp/build/macosx/obj/Gen.o src/slice2cpp/build/macosx/obj/Main.o -mmacosx-version-min=12.00 lib/libSlice.a lib/libIceUtil.a   -L/opt/homebrew/opt/mcpp/lib /opt/homebrew/opt/mcpp/lib/libmcpp.a
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[2](main.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[3](directive.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[4](eval.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[5](expand.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[6](mbchar.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[7](support.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
ld: warning: object file (/opt/homebrew/opt/mcpp/lib/libmcpp.a[8](system.o)) was built for newer 'macOS' version (14.0) than being linked (12.0)
```